solo-chains/runtime/dancelight/src/tests/mod.rs

```diff
@@ -68,3 +73,116 @@ fn check_treasury_pallet_id() {
         dancelight_runtime_constants::TREASURY_PALLET_ID
     );
 }
+
+#[test]
```

Comment:
I dont think this test should belong to mod.rs, specially when we dont have any test (that I know of) in that file. Either move it to its own test file or put it in integration_tests.rs

Reply:
*fixed*